File: interface/src/main/java/cn/itcast/core/service/UserService_ZhangFeng.java
package cn.itcast.core.service;


import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.user.User;

import java.util.List;
/*
张锋
 */
public interface UserService_ZhangFeng {

    /*
    张锋
     */
    public List<User>findAll();

        /*
    张锋
     */
    void add(User user);
    /*
    张锋
     */
    User findOne(Long id);
    /*
    张锋
     */
    void update(User user);
    /*
    张锋
     */
    void delete(Long[] ids);
    /*
    张锋
     */
    PageResult findPage(Integer page, Integer rows, User user);

}

File: web_manager/src/main/webapp/js/service/userService_zhangfeng.js
// 定义服务层:
app.service("userService",function($http){
	this.findAll = function(){
		return $http.get("../user/findAll.do");
	}
    this.count = function(){
        return $http.get("../user/count.do");
    }
	
	this.findByPage = function(page,rows){
		return $http.get("../user/findByPage.do?page="+page+"&rows="+rows);
	}
	
	this.save = function(entity){
		return $http.post("../user/add.do",entity);
	}
	
	this.update=function(entity){
		return $http.post("../user/update.do",entity);
	}
	
	this.findById=function(id){
		return $http.get("../user/findOne.do?id="+id);
	}
	
	this.dele = function(ids){
		return $http.get("../user/delete.do?ids="+ids);
	}
	
	this.search = function(page,rows,searchEntity){
		return $http.post("../user/search.do?page="+page+"&rows="+rows,searchEntity);
	}
	
	this.selectOptionList = function(){
		return $http.get("../user/selectOptionList.do");
	}
    //审核状态
    //修改
    this.updateStatus = function(ids,status){
        return $http.get('../user/updateStatus.do?ids='+ids+"&status="+status);
    }
});